src/lib/get-wordpress-version-url.ts

```diff
@@ -1,7 +1,12 @@
+import { isWordPressDevVersion } from 'src/lib/version-utils';
 import { DEFAULT_WORDPRESS_VERSION } from 'vendor/wp-now/src/constants';
 import { isValidWordPressVersion } from 'vendor/wp-now/src/wp-playground-wordpress/is-valid-wordpress-version';
 
 export function getWordPressVersionUrl( version = DEFAULT_WORDPRESS_VERSION ) {
+	if ( isWordPressDevVersion( version ) ) {
+		return 'https://wordpress.org/nightly-builds/wordpress-latest.zip';
+	}
```

Comment:
I'm a bit worried that when selecting a beta version we actually download a nightly version. This is not the case right now, but it could happen if the API returns a `6.8-beta3-600046`.

It would be better if we identify a `nightly` with that version name.